enroll/certificates.go

```diff
@@ -1,22 +1,39 @@
 package enroll
 
 import (
+	"crypto/x509"
+	"encoding/pem"
 	"errors"
 	"golang.org/x/crypto/pkcs12"
 	"io/ioutil"
 )
 
-const PushTopicASN1 string = "0.9.2342.19200300.100.1.1"
+const (
+	certificatePEMBlockType string = "CERTIFICATE"
+	PushTopicASN1                  = "0.9.2342.19200300.100.1.1"
```

Comment:
The PushTopic could be changed to an oid var, and then you can use the correct `.Equals` function from `asn1`. 

The change would be something like `var oidASN1Topic = asn1.ObjectIdentifier{0,9,2342,...}` and then you can compare with https://golang.org/pkg/encoding/asn1/#ObjectIdentifier.Equal
I know the const was there from the previous PR, but I think it would be good to clean up :)